app/routes/customer_routes.py

```diff
@@ -0,0 +1,74 @@
+from app import db
+from flask import Blueprint, jsonify, request
+from app.models.customer import Customer
+from app.models.rental import Rental
+from app.models.video import Video
+from app.models.helper import invalid_cust_data, invalid_customer
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+
+@customers_bp.route("", methods=["POST"])
+def post_customer():
+    request_body = request.get_json()
+    invalid_response = invalid_cust_data(request_body)
+    if not invalid_response:
+        new_customer = Customer(
+            name=request_body["name"],
+            postal_code=request_body["postal_code"],
+            phone=request_body["phone"]
+        )
+        db.session.add(new_customer)
+        db.session.commit()
+        return jsonify(new_customer.to_json()), 201
+    return jsonify(invalid_response), 400
```

Comment:
Consider reordering this check into a guard clause as follows, rather than having the error behavior isolated at the end.

```python
    if invalid_response:
        return jsonify(invalid_response), 400

    new_customer = Customer(
        name=request_body["name"],
        postal_code=request_body["postal_code"],
        phone=request_body["phone"]
    )
    db.session.add(new_customer)
    db.session.commit()
    return jsonify(new_customer.to_json()), 201
```